### PR TITLE
fix: Disable unlock when there is no ckb.

### PR DESCRIPTION
**Pull request by yanguoyu**
Refer to https://github.com/Magickbase/neuron-public-issues/issues/350
1. Disabled withdraw when there is no ckb balance
image

2. Enable withdraw when there exists ckb balance

image

### CODE REVIEW

**Review by devchenyan — APPROVED** [review]

File: packages/neuron-ui/src/components/NervosDAO/index.tsx [hunk 1]
```diff
@@ -179,6 +179,7 @@ const NervosDAO = () => {
                 genesisBlockTimestamp,
                 connectionStatus,
                 isPrivacyMode,
+                hasCkbBalance: !!wallet.balance && wallet.balance !== '0',
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/NervosDAO/index.tsx:182]: Why not use `+wallet.balance > 0`

File: packages/neuron-ui/src/locales/en.json [hunk 2]
```diff
@@ -689,7 +689,8 @@
         "compensated-period": "Compensation Period",
         "days-hours": "{{days}} days {{hours}} hours",
         "no-deposit": "No Deposit",
-        "no-completed": "No Completed"
+        "no-completed": "No Completed",
+        "no-ckb": "No other CKB to pay for transaction fee that unlock DAO"
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/en.json:693]: `No other CKB` is hard to understand, how about `Balance is not enough for transaction fee to unlock the deposit`

File: packages/neuron-ui/src/components/NervosDAO/index.tsx (hunk 1 above)
Comment (yanguoyu) [reply packages/neuron-ui/src/components/NervosDAO/index.tsx]: The `wallet.balance` should transform to BigInt, but for comparison to 0, loss of precision doesn't matter.
I have changed it to `+wallet.balance > 0`

File: packages/neuron-ui/src/locales/zh.json [hunk 3]
```diff
@@ -682,7 +682,8 @@
         "compensated-period": "补贴时间",
         "days-hours": "{{days}} 天 {{hours}} 小时",
         "no-deposit": "没有存入记录",
-        "no-completed": "没有解锁记录"
+        "no-completed": "没有解锁记录",
+        "insufficient-balance-to-unlock": "没有余额支付交易费用以解锁 DAO 存款"
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/zh.json:686]:
```suggestion
        "insufficient-balance-to-unlock": "没有足够余额支付交易费用以解锁 DAO 存款"
```

File: packages/neuron-ui/src/locales/zh-tw.json [hunk 4]
```diff
@@ -682,7 +682,8 @@
         "compensated-period": "補貼時間",
         "days-hours": "{{days}} 天 {{hours}} 小時",
         "no-deposit": "沒有存入記錄",
-        "no-completed": "沒有解鎖記錄"
+        "no-completed": "沒有解鎖記錄",
+        "insufficient-balance-to-unlock": "沒有余額支付交易費用以解鎖 DAO 存款"
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/zh-tw.json:686]:
```suggestion
        "insufficient-balance-to-unlock": "沒有足夠余額支付交易費用以解鎖 DAO 存款"
```

**Review by Keith-CY — APPROVED** [review]

**Review by homura — APPROVED** [review]

**Review by silySuper — APPROVED** [review]

**Pull request merged**